### DISCUSSION TITLE
I run the application but get a blank white screen.

### DISCUSSION THREAD

**Discussion by SUVENDU01** [NASA-SW-VnV/fret#67] [Q&A]
I followed this instructions
https://github.com/NASA-SW-VnV/fret/blob/master/fret-electron/docs/_media/installingFRET/installationInstructions.md

after setup the environment
koushikghosh@Koushiks-MacBook-Air fret-master % npm --version
7.7.6
koushikghosh@Koushiks-MacBook-Air fret-master % node --version
v15.14.0
koushikghosh@Koushiks-MacBook-Air fret-master % python3 --version
Python 3.10.11

I run the application but get a blank white screen.

I'm attaching the setup and warning log along with the application screenshot.
[installation log.txt](https://github.com/NASA-SW-VnV/fret/files/13523083/installation.log.txt)
![Screenshot 2023-12-01 at 12 50 47 AM](https://github.com/NASA-SW-VnV/fret/assets/40097541/6a8e0e48-3796-410e-8866-ad2a505dc3ef)

**Comment by anmavrid** [comment]
Hi @SUVENDU01,

Thanks or your message.

Does your Mac have an M1 chip?

Anastasia
